src/nginxconfig/i18n/en/templates/global_sections/tools.js

```diff
@@ -37,9 +37,17 @@ export default {
     resetGlobalConfig: 'Reset global config',
     resetAllDomains: 'Reset all domains',
     removeAllDomains: 'Remove all domains',
+    removeAllDomainsConfig: 'Reset all domain configs',
     resetDomainConfig: 'Reset domain config',
     removeDomain: 'Remove domain',
     yesImSure: 'Yes, I\'m sure',
     noCancel: 'No, cancel',
     tools: 'Tools',
+    resetGlobalConfigBody: 'Are you sure you want to reset all configuration options in the global config section?',
+    removeAllDomainsConfigBody: 'Are you sure you want to reset the configuration of ALL domains?',
```

Comment:
```suggestion
    resetAllDomainsConfigBody: 'Are you sure you want to reset the configuration of ALL domains?',
```
Use `reset` to be clearer